blog/_posts/2016-09-30-Thank-you-all-from-melbourne.md

```diff
@@ -0,0 +1,50 @@
+---
+title: The end of an amazing Summer(winter) Of Code!!
+layout: post
+created_at: Sun sept 30 2016
+permalink: blog/2016-09-30-amazing end of the winter
+current: blog
+author: ramya
+categories:
```

Comment:
As pointed out in the [contributing guide](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#creating-a-new-blog-post) we still need a twitter line for the author name to display properly on the blog. Can you add that? See past blog posts for examples.